### PR TITLE
bugfix/3 - Add event listeners

### PR DESCRIPTION
**Pull request by magnifiq**
After the addition of an event listener

### CODE REVIEW

File: form.js [hunk 1]
```diff
@@ -1,26 +1,44 @@
...
+function appSubEl(parent, child){
+  return parent.appendChild(child);
+}
 
-  const inputPassword = document.createElement("input");
-  inputPassword.type = "text";
-  inputPassword.placeholder = "Enter your password";
+function getEl(target){
+  return document.getElementById(target).value;
+}
```
Comment (igorTT) [inline form.js:18]: function returns not the element, but its value. Rename to getElementValue.

File: form.js [hunk 2]
```diff
@@ -1,26 +1,44 @@
...
 
-  const btnSub = document.createElement("button");
-  btnSub.type = "submit";
-  btnSub.textContent = "Submit";
+function onSubmit(e){
+  e.preventDefault();
+  console.log(getEl("email"));
+  console.log(getEl("password"));
```
Comment (igorTT) [inline form.js:23]: You already have access to these two elements -> inputEmail and inputPassword

**Review by igorTT — CHANGES_REQUESTED** [review]
please check target branch to reduce changes.

**Review by igorTT — CHANGES_REQUESTED** [review]
The pull request should contain only the functionality described in the issue (add event listeners)

**Comment by magnifiq** [comment]
ok, I understood, but I added the local storage and a loading state here. Do I need to add this changes to the solved_first branch?

**Comment by igorTT** [comment]
> ok, I understood, but I added the local storage and a loading state here. Do I need to add this changes to the solved_first branch?

This pull request should contain only changes related to the current issue (add event listeners). All other issues should be solved in separate branches.

**Review by igorTT — APPROVED** [review]

**Pull request merged**